README.md

```
@@ -157,33 +157,52 @@ dotnet test --verbosity normal
 
 The tests cover:
 
-- Success and failure result creation
-- Type safety with generic results
-- Error handling patterns
-- Complex type support
+-   Success and failure result creation
+-   Type safety with generic results
```

markdownlint-cli2 (0.18.1)

Spaces after list markers
Expected: 3; Actual: 2

(MD030, list-marker-space)